pkgs/libxml2/2.12.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
```

Comment:
optional, nit:  Should we consider the use of `nixfmt-rfc-style`?

Reply:
Probably. At first I decided against it to have it match Nixpkgs but then I simplified the expression so that ship has sailed.